system/lib/libc/musl/src/string/memcpy.c

```diff
@@ -2,11 +2,76 @@
 #include <stdint.h>
 #include <endian.h>
 
+#ifdef __EMSCRIPTEN__
+// An external JS implementation that is efficient for very large copies, using
+// HEAPU8.set()
+extern void *emscripten_memcpy_big(void *restrict dest, const void *restrict src, size_t n);
+#endif
+
 void *memcpy(void *restrict dest, const void *restrict src, size_t n)
 {
 	unsigned char *d = dest;
 	const unsigned char *s = src;
 
+#ifdef __EMSCRIPTEN__
+
+  unsigned char *aligned_d_end;
```

Comment:
Apparently this file was using tabs before, and now it's mixed tabs+spaces. Probably convert to all-tabs to minimize upstream diffs? Either way it shouldn't be mixed.

Reply:
Thanks, fixed to tabs.